src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -116,6 +120,12 @@
     /** Fully-qualified {@link Deprecated Deprecated} annotation name. */
     private static final String FQ_DEPRECATED = "java.lang." + DEPRECATED;
 
+    /** List of token types to find parent of. */
+    private static final List<Integer> TYPES_ARRAY_LIST = new ArrayList<>(Arrays.asList(
```

Comment:
please use Set collection for quicker search.

Reply:
Fixed.